.github/workflows/build-deploy.yml

```diff
@@ -106,7 +106,7 @@ jobs:
           CLUSTER_NAME: ${{ vars.CLUSTER_NAME  }}
           ECR_REGISTRY: ${{ secrets.ECR_REGISTRY  }}
           ECR_REPO_NAME: ${{ env.ECR_REPO_NAME }}
-          ENDPOINT: ${{ vars.APP_NAME }}.${{ secrets.DOMAIN_NAME }}
+          ENDPOINT: ${{ vars.APP_NAME }}-${{ vars.ENV }}.${{ secrets.DOMAIN_NAME }}
```

Comment:
A conditional might have been better here, since in prod this will fail. Could be something like:
```
${{ vars.ENV == 'staging' && '-staging' || '' ) }}
```